src/test/java/htsjdk/samtools/SAMFileWriterFactoryTest.java

```diff
@@ -220,6 +250,28 @@ private void verifyWriterOutput(File outputFile, ReferenceSource refSource, int
         Assert.assertTrue(count == nRecs);
     }
 
+    private void verifyWriterOutput(Path output, ReferenceSource refSource, int nRecs, boolean verifySupplementalFiles) throws IOException {
+        if (verifySupplementalFiles) {
+            final Path index = SamFiles.findIndex(output);
+            final Path md5File = IOUtil.addExtension(output, ".md5");
+            Assert.assertTrue(Files.size(index) > 0);
```

Comment:
The text is there twice, but one of them uses a File and other other uses a Path. Not much there to deduplicate.